src/components/live/Landing.jsx

```diff
@@ -1,5 +1,16 @@
 const Landing = () => {
-  return <div className="flex items-center justify-center">Landing</div>;
+  return (
+    <div className="flex justify-evenly bg-[#021334] p-6 font-semibold text-white">
+      <div className="p-2">HOME</div>
+      <div className="p-2">SCHEDULE</div>
+
+      <div className="p-2">ABOUT US</div>
+
+      <div className="border-white p-2 font-goodtimes shadow-custom-white">
+        LOGIN
```

Comment:
https://github.com/citrushack/cutiehack2024/pull/10/files#diff-9759e97dc8e39d64a12d5187a1ede3fa4f570f5b585437bf0207bd2d8d544c4c
look at the logic in creating sign in and add it to the navbar